packages/app/src/cli/services/app/config/link.ts

```diff
@@ -211,3 +218,24 @@ const getAccessScopes = (appConfiguration: AppConfiguration, remoteApp: Organiza
     }
   }
 }
+
+export async function getRemoteAppConfig(apiKey: string): Promise<Partial<CurrentAppConfiguration>> {
+  const token = await ensureAuthenticatedPartners()
+  const remoteSpecifications = await fetchAppExtensionRegistrations({token, apiKey})
+  const appAccessModule = remoteSpecifications.app.extensionRegistrations.find((extension) => {
+    return extension.type.toLowerCase() === APP_ACCESS_IDENTIFIER
+  })
+  const appConfig: Partial<CurrentAppConfiguration> = {}
+
+  if (appAccessModule?.activeVersion?.config) {
+    try {
+      const appAccessConfig = JSON.parse(appAccessModule.activeVersion.config)
```

Comment:
@isaacroldan, what's the context behind `activeVersion.config` being a string that needs to be `JSON.parse`'d?

Reply:
I don't think that's required

Reply:
It contains the "deploy" payload for the current version of the module

Reply:
nop, easier schema maybe? I don't know